models/dao/user-survey.dao.js

```diff
@@ -29,6 +29,29 @@ const comparators = {
 
         return !_.isEqual(answers, ruleAnswers);
     },
+    'in-date-range': function (answers, ruleAnswers) {
+        if (!(answers && answers.length)) {
+            return false;
+        }
+        if (answers.length !== 1 || ruleAnswers.length !== 1) {
+            return false; // not yet implemented
```

Comment:
Is there an issue number we can reference here for what the implementation will look like?

Reply:
RR-962 is added.